apartmenttherapy/oxygen-lib-ui
# html entities correction in metadata

#### What does this PR do & why?
- run all metadata through function that cleans out any html entities

#### How can this be manually tested?
- use oxygen-graphql pr: https://github.com/apartmenttherapy/oxygen-gql/pull/85
- `yarn install` un oxygen-lib-ui
- run dev on K
- go to http://localhost:8080/mary-berry-home-rental-cottage-kitchen-261598
- compare to: https://oxygen-kitchn-dot-gen-prod.appspot.com/mary-berry-home-rental-cottage-kitchen-261598
- see that the pr's title and various other meta like description, opengraph and twitter do not have html entities

#### Any questions or concerns about this change?

_(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)_

#### Deployment Notes

_(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)_

#### Link to story card and other documentation
https://app.asana.com/0/inbox/625204677761211/814143913874421/852101202408289

#### GIF

#### Have you remembered to...?
